src/state/store/initialState.js

```diff
@@ -0,0 +1,5 @@
+const initialState = {
+	newsFeed: [],
```

Comment:
since you have an empty array here, then you can probably skip the conditional in your component that checking if there is something in it. It is entirely fine to try to iterate over an empty array. 

Also, I would add `spacificArticle` here as an empty object. That will also save you a few lines of code in your components.

Reply:
Added specificArticle to initialState